src/utils/files.ts

```
@@ -16,3 +16,23 @@
+
+export async function getFilesInDirectoryRecursive(dir: string): Promise<string[]> {
+  const files = await fs.promises.readdir(dir)
+  const filePaths: string[] = []
+
+  for (const file of files) {
+    const fullPath = join(dir, file)
+    const stat = await fs.promises.stat(fullPath)
+
+    if (stat.isFile()) {
+      filePaths.push(fullPath)
+    } else if (stat.isDirectory()) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(@stylistic/brace-style)